src/components/ChartContainer/D3Container.js

```diff
@@ -22,6 +22,38 @@ function D3Container({ dashboardState, dashboardActions, store }) {
   const [byLineMeasure, setByLineMeasure] = useState('');
   const [byLineDisplayData, setByLineDisplayData] = useState([]);
 
+  const colorArray = [
```

Comment:
Should be singlequote. And because it's pretty static, you could move it just outside of `D3Container`'s declaration.

Reply:
Corrected.